variables.tf

```diff
@@ -207,3 +207,9 @@ variable "tags" {
   description = "A map of tags to add to the resources"
   default     = {}
 }
+
+variable "ssl_policy" {
+  type        = "string"
+  description = "SSL policy applied to an SSL enabled ALB, see https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html"
+  default     = "ELBSecurityPolicy-2015-05"
```

Comment:
Why do we set this as default and not the recommended: ELBSecurityPolicy-2016-08 https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html

Reply:
Update the default to the latest version, add comment to changelog.